### ISSUE TITLE
WAVE Flags Broken ARIA reference in Front End Highlighting when logged in

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#285]
This is the code it flags:
```
<div id="edac-highlight-panel-description" class="edac-highlight-panel-description" role="dialog" aria-labelledby="edac-highlight-panel-description-title" tabindex="0">
ARIA label
<button id="edac-highlight-panel-controls-close" class="edac-highlight-panel-description-close edac-highlight-panel-controls-close" aria-label="Close">
×
</button>
<div class="edac-highlight-panel-description-title"></div>
<div class="edac-highlight-panel-description-content"></div>
<div id="edac-highlight-panel-description-code" class="edac-highlight-panel-description-code">
<code></code>
</div>
</div>
```
I think this is mostly a false positive, but it seems messy and it's safe to assume our users will see this in WAVE.
